entry_types/scrolled/lib/pageflow_scrolled/seeds.rb

```diff
@@ -25,6 +25,11 @@ def sample_scrolled_entry(attributes)
                                         **attributes.except(:sections)) do |created_entry|
           created_entry.theming = attributes.fetch(:account).default_theming
 
+          features_configuration = created_entry.features_configuration
+                                                .merge('page_types_with_file_types' => false,
+                                                       'page_type.linkmap_page' => false)
+          created_entry.features_configuration = features_configuration
+
```

Comment:
Let's put this in the host application since it relies on a special temporary setup we have created there.